### PR TITLE
Feat/substraction

### PR DESCRIPTION
**Pull request by malvarez96-afk**
Addition of substraction function

### CODE REVIEW

File: tests/test_matrix.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from matrix import Matrix
+
```
Comment (diego-padron777) [inline tests/test_matrix.py:2]: You forgot to include a test with a empty matrixes

**Review by diego-padron777 — APPROVED** [review]
Looks good to me

**Review by sebasmtzo — COMMENTED** [review]
good job mateo!

**Review by sebasmtzo — APPROVED** [review]
good job mateo!
